### PR TITLE
Bump min compatibility to PHP 7.1.3 and PS 1.7.7, module version to 3.0.0

### PR DESCRIPTION
**Pull request by leemyongpakvn**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please check the related issue.
| Type?         | improvement
| BC breaks?    | Does it break backward compatibility? yes
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes PrestaShop/Prestashop#32841
| How to test?  | CI  🟢. Module works as usual on PS 1.7.7 and newer versions.

### CODE REVIEW

**Comment by leemyongpakvn** [comment]
@kpodemski We still face `Clear previous module` failure here.

**Comment by kpodemski** [comment]
@leemyongpakvn this time it's different I think 🤔 it looks like it has been deleted, and we try to remove something that doesn't exist

File: tests/php/phpstan.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ docker run -tid --rm -v ps-volume:/var/www/html --name temp-ps prestashop/presta
 # Clear previous instance of the module in the PrestaShop volume
 echo "Clear previous module"
 
-docker exec -t temp-ps rm -rf /var/www/html/modules/ps_emailalerts
+docker exec -t --user root temp-ps sh -c 'find /var/www/html/modules/ps_facetedsearch -type f -exec rm {} +'
```
Comment (kpodemski) [inline tests/php/phpstan.sh:17]:
```suggestion
docker exec -t --user root temp-ps sh -c 'find /var/www/html/modules/ps_emailalerts -type f -exec rm {} +'
```

:)

File: tests/php/phpstan.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ docker run -tid --rm -v ps-volume:/var/www/html --name temp-ps prestashop/presta
 # Clear previous instance of the module in the PrestaShop volume
 echo "Clear previous module"
 
-docker exec -t temp-ps rm -rf /var/www/html/modules/ps_emailalerts
+docker exec -t --user root temp-ps sh -c 'find /var/www/html/modules/ps_emailalerts -type f -exec rm {} +'
```
Comment (kpodemski) [inline tests/php/phpstan.sh:17]:
```suggestion
docker exec -t --user root temp-ps sh -c 'if [ -d "/var/www/html/modules/ps_emailalerts" ]; then find /var/www/html/modules/ps_emailalerts -type f -exec rm {} +; fi'
```

something like this maybe?

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thank you @leemyongpakvn - since it has no functional changes, I'm merging it

**Pull request merged**